src/twisted/words/protocols/irc.py

```diff
@@ -1238,6 +1238,9 @@ class IRCClient(basic.LineReceiver):
     _heartbeat = None
     heartbeatInterval = 120
 
+    decode_codecs = ["utf-8"]  # IRC protocol doesn't specify encoding
+    decode_fallback_errorhandling = "replace"  # see bytes.decode
```

Comment:
The names must match the Twisted convention: `decodeCodecs`, `decodeFallbackErrors`.

You also have to add `@ivar` annotations for them to the class docstring.